### ISSUE TITLE
Amazon Fire TV Stick 4k

### ISSUE THREAD

**Issue by GOW2017** [TailsxKyuubi/inoffizielle-AoD-App#44]
Hey ich bin es nochmal und wollte mal ein Statusupdate geben, 

Es ist anscheinend so, dass sowohl ihr als auch die Amazon Mitarbeiter recht geahabt bezüglich des Updates. 
Das neue Update kam nämlich (neue Oberfläche und alles drum und dran) tatsächlich, aber ohne Fire OS7.  
Deswegen ist der Fullscreen Bug immernoch da. 

Ist es möglich die App auch auf Fire OS 6.2 kompatibel zu machen?

**Comment by TailsxKyuubi** [comment]
Ich könnte es versuchen, aber das geht nicht solange ich mich bei der TV Entwicklung die ganze Zeit im Blindflug befinde. Kurz gesagt, ich habe keine Möglichkeit herauszufinden was überhaupt das Problem ist, da ich selbst weder einen richtigen FireTV noch Android TV habe.

**Comment by TailsxKyuubi** [comment]
@GOW2017 Wenn mir ein entsprechender Stick zur Verfügung gestellt wird, kann ich mir das gerne im Detail anschauen, allerdings sehe ich nicht ein, dass ich für die App mir alle möglichen Geräte kaufen soll, nur damit ich den Support der App dafür gewährleisten kann und ansonsten diese für mich nutzlos sind.

**Comment by RabitOne** [comment]
Brauchst du noch einen Fire TV stick? Würde einen Sponsoren.
Finde es toll das du dir die Arbeit machst, mache bitte weiter so ^-^

**Comment by TailsxKyuubi** [comment]
Haben tu ich,, nach wie vor keinen. Wenn du einen sponsoren möchtest, könnte ich mehr Fokus auf den TV Support legen

**Comment by RabitOne** [comment]
Ja, gerne, habe nämlich auch Probleme mit der Auflösung. Da meine Internet nicht wirklich viel Leistung bring, muss ich die Auflösung runtersetzen, aber dadurch bekomme ich auch ein kleineres Videobild mit Greenscreen. Deswegen würde ich mich sehr darüber freuen, wenn du noch weiter daran  arbeiten würdest ^^
Jetzt ist nur die Frage, gibst du mir eine Adresse wo ich den Fire TV stick 4k hinschicken soll oder solle ich dir das Geld dafür per Paypal senden und du holst ihn dir selbst

**Comment by TailsxKyuubi** [comment]
Ich schicke dir am besten einen Link mit einer Amazon Wunschliste, da kannst du das dann direkt zu mir schicken lassen.

**Comment by RabitOne** [comment]
Jup, das geht auch 😁

**Comment by TailsxKyuubi** [comment]
https://www.amazon.de/hz/wishlist/ls/1LV7308UL3X7E

**Comment by RabitOne** [comment]
Okay, bestellt 😊

**Comment by TailsxKyuubi** [comment]
Habe mir das gestern mal das Problem detail angeschaut und eine simple Lösung ist leider nicht möglich, ich habe mal Amazon kontaktiert und hoffe dass das Problem bald gelöst werden kann.
Durch diesen Bug habe ich lustigerweise einen anderen Bug endeckt, der direkt behoben wurde

**Comment by TailsxKyuubi** [comment]
Kleiner Zwischenstand: Der Fall wurde jetzt direkt an die Entwickler von Amazon weitergegeben und diese schauen sich genauer an was da zwischen dem Player und FireOS 6 schief läuft

**Comment by RabitOne** [comment]
Okay, dann bleibt zu hoffen, daß die auch was daran ändern

**Comment by sladexos** [comment]
Also ich hatte auch den Fullscreen-Bug (Bild nur in der linken oberen Ecke, der Rest war grün). Nachdem ich in den Player-Einstellungen auf 1080+ umgestellt habe und die "eigene Lautstärke..." verwenden deaktiviert habe, funktioniert es ohne Probleme. Sowohl App als auch FireTV Stick sind auf dem neuesten Stand.

Ich bin zufrieden und schicke ein großes Dankeschön an den Entwickler :-)

**Comment by TailsxKyuubi** [comment]
Vielen Dank für den Hinweis. Bei einem schnellen Test mit der 0.8 konnte ich da ein verändertes verhalten bestätigen. ich werde das die tage genauer untersuchen und dann einen fix für die version 0.8 anfertigen

**Comment by TailsxKyuubi** [comment]
ich habe das bei mir soweit möglich getestet und konnte nicht nachweisen dass dies die Problemlösung ist. Ich vermute mal, dass dein Fire Stick sich auf 1080p eingestellt hat und dementsprechend das bei dir dann alles ausfüllt.

**Comment by TailsxKyuubi** [comment]
Aktuell der einzige workaround ist, den stick auf 1080p zu stellen und in der app das gleiche zu tun

**Comment by TailsxKyuubi** [comment]
Ich habe einen weg gefunden das Problem zu lösen. Dafür wird vom ExoPlayer auf Vlc umgestellt. In kürze wird der entsprechende code hier auf github landen und auf dem code mirror dann auch die version automatisch kompiliert
